openshift/system/config/backend.yml

```diff
@@ -1,13 +1,7 @@
-base: &default
-  host: backend-listener:3000
-  public_url: <%= ENV.fetch("BACKEND_ROUTE") { "https://backend-#{ENV['TENANT_NAME']}.#{ENV['THREESCALE_SUPERDOMAIN']}" } %>
-
 production:
-  <<: *default
-  secure: false
-  persistent: true
-
-preview:
-  <<: *default
-  secure: false
-  persistent: true
+  url: <%= ENV.fetch('BACKEND_URL', 'http://backend-listener:3000') %>
+  public_url: <%= ENV.fetch('BACKEND_ROUTE') { "https://backend-#{ENV['TENANT_NAME']}.#{ENV['THREESCALE_SUPERDOMAIN']}" } %>
+  persistent: <%= ENV.fetch('BACKEND_PERSISTENT', '1') == '1' %>
+  max_tries: <%= ENV.fetch('BACKEND_MAX_TRIES', 5) %>
+  open_timeout: <%= ENV.fetch('BACKEND_OPEN_TIMEOUT', 3) %>
```

Comment:
`open_timeout` and `timeout` defaults to 3 in the code https://github.com/3scale/porta/blob/e2baf42/app/lib/backend_client/request.rb#L46